data.sql

```diff
@@ -16,3 +16,33 @@ VALUES ('Pikachu', '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', '2017-05-12', 5, TRUE, 11.00);
 
+-- Add a column 'species' of type string to the 'animals' table.
+ALTER TABLE animals ADD COLUMN species VARCHAR(255);
+
```

Comment:
As mentioned by the previous reviewer and for the project requirements, kindly ensure that everything that creates or change the tables structure should be saved in the `schema.sql` file, as reason the above query to be moved in  `schema.sql` file,